src/Integration.UnitTests/LocalServices/SolutionBindingInformationProviderTests.cs

```diff
@@ -65,143 +66,117 @@ public void SolutionBindingInformationProvider_ArgCheck()
         }
 
         [TestMethod]
-        public void SolutionBindingInformationProvider_GetBoundProjects_SolutionNotBound_StandaloneMode()
+        public void SolutionBindingInformationProvider_GetUnboundProjects_SolutionBound_EmptyFilteredProjects()
```

Comment:
I deleted the first two tests since there are already _unbound_ equivalents for them (father down in the file).

However, the _GetBoundProjects_ tests were more comprehensive than the tests for _GetUnBound_methods, so I've kept the rest of those tests but changed them to call _GetUnboundProjects_, and changed the assertions to look for the inverse set of projects.